builtin/providers/azurerm/resource_arm_template_deployment.go

```diff
@@ -155,20 +155,44 @@ func resourceArmTemplateDeploymentRead(d *schema.ResourceData, meta interface{})
 	if resp.Properties.Outputs != nil && len(*resp.Properties.Outputs) > 0 {
 		outputs = make(map[string]string)
 		for key, output := range *resp.Properties.Outputs {
+			log.Printf("[DEBUG] Processing deployment output %s", key)
 			outputMap := output.(map[string]interface{})
 			outputValue, ok := outputMap["value"]
 			if !ok {
 				// No value
 				continue
 			}
+			outputType := outputMap["type"]
+			var outputStringValue string
+
+			switch outputType {
+			case "Bool":
+				if outputValue == false {
+					outputStringValue = "0"
+				} else if outputValue == true {
+					outputStringValue = "1"
+				} else {
+					return fmt.Errorf("Invalid value %s for boolean output %s", outputValue, key)
+				}
```

Comment:
BTW, most of the code I see does not add [ERROR] to error messages, just to logging?  What's the standard?  Presumably [ERROR]'s not needed since it already shows up as an error?